include/temoto_robot_manager/robot_manager_interface.h

```diff
@@ -269,12 +269,11 @@ class RobotManagerInterface : public temoto_core::BaseSubsystem
   }
 
   void navigationGoal(const std::string& robot_name
-  , const std::string& reference_frame
   , const geometry_msgs::PoseStamped& pose)
   {
-    temoto_robot_manager::RobotNavigationGoal msg; 
-    msg.request.reference_frame = reference_frame;
+    temoto_robot_manager::RobotNavigationGoal msg;
     msg.request.target_pose = pose;
+    msg.request.target_pose.header.frame_id = (pose.header.frame_id.empty()) ? "map" : pose.header.frame_id;
```

Comment:
I think it's not a good idea to have a default frame_id inside this or any Robot Manager service call. This may lead to undesired behaviors at some point. Rather just throw an "Unspecified frame_id" error.